### DISCUSSION TITLE
jvex: Java types for OpenVEX documents

### DISCUSSION THREAD

**Discussion by p-ssanders** [openvex/community#18] [Show and tell]
Hey 👋 I love the work you're doing with OpenVEX. I made a Java implementation of the latest spec (v0.2.0). Hopefully this can enhance the list of tools available for authors and consumers using Java.

Appreciate any and all feedback!

https://github.com/p-ssanders/jvex

Thanks,
Sam

**Comment by puerco** [comment]
This is awesome @p-ssanders , I'll add it to the known implementations. I'm presenting on OpenSSF day and I'll give a shoutout to your implementation. Thank you!!

**Reply by p-ssanders** [reply]
Perfect, thanks. And good luck on your talk!
